var/spack/repos/builtin/packages/hsa-rocr-dev/package.py

```diff
@@ -4,6 +4,8 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 
+import os.path
```

Comment:
```suggestion
import os
```

From the docs:

> DESCRIPTION
>    Instead of importing this module directly, import os and refer to
>    this module as os.path.